test/Microsoft.Management.Infrastructure.Tests/NativeTests/MOFDeserializerTests.cs

```diff
@@ -23,13 +23,15 @@ public MOFDeserializerTests() : base(MI_SerializationFormat.MOF)
         {
         }
 
+/* @TODO Fix me later 
```

Comment:
file an issue and link

Reply:
Or maybe list the lines  in the issue

Reply:
https://github.com/PowerShell/MMI/issues/38